composer.json

```diff
@@ -21,7 +21,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.0.x-dev"
+            "dev-master": "3.0.x-dev"
```

Comment:
Shouldn't the alias be 2.0.x-dev here? (as explained in https://github.com/php-fig/fig-standards/pull/1215)

Reply:
Yep - fixed now.